src/bitcoin-cli.cpp

```diff
@@ -470,20 +455,30 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (networkinfo["version"].getInt<int>() < 209900) {
             throw std::runtime_error("-netinfo requires bitcoind server to be running v0.21.0 and up");
         }
+        // Initialize the peer-count-by-network-type map with all the reachable networks (always display)
+        for (const UniValue& network : networkinfo["networks"].getValues()) {
+            if (!network["reachable"].get_bool()) {
+                continue;
+            }
+            const std::string network_name{network["name"].get_str()};
+            m_counts[network_name];
+        }
+
         const int64_t time_now{TicksSinceEpoch<std::chrono::seconds>(CliClock::now())};
 
         // Count peer connection totals, and if DetailsRequested(), store peer data in a vector of structs.
         for (const UniValue& peer : batch[ID_PEERINFO]["result"].getValues()) {
             const std::string network{peer["network"].get_str()};
-            const int8_t network_id{NetworkStringToId(network)};
-            if (network_id == UNKNOWN_NETWORK) continue;
             const bool is_outbound{!peer["inbound"].get_bool()};
             const bool is_tx_relay{peer["relaytxes"].isNull() ? true : peer["relaytxes"].get_bool()};
             const std::string conn_type{peer["connection_type"].get_str()};
-            ++m_counts.at(is_outbound).at(network_id);      // in/out by network
-            ++m_counts.at(is_outbound).at(NETWORKS.size()); // in/out overall
-            ++m_counts.at(2).at(network_id);                // total by network
-            ++m_counts.at(2).at(NETWORKS.size());           // total overall
+            // If peer network is not reachable, insert into map (only display if used)
+            m_counts[network];
+            if (is_outbound) {
+                ++m_counts.find(network)->second.second;
+            } else {
+                ++m_counts.find(network)->second.first;
+            }
```

Comment:
```suggestion
            ++std::get<is_outbound ? 1 : 0>(m_counts[network]);
```

Reply:
I didn't think you can use a variable in `std::get< ? >` ?

```
bitcoin-cli.cpp:476:15: error: no matching function for call to 'get'
 ++std::get<is_outbound ? 1 : 0>(m_counts[network]);
```

https://stackoverflow.com/questions/43621098/c-stdgetvariable-fails